sh/publish.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I prefer using `#!/usr/bin/env bash` as it will load whatever version of bash appears first in your $PATH rather than the specific one located at `/bin/bash`.